tests/testthat/debug.csv

```diff
@@ -0,0 +1,101 @@
+﻿,contNormal,contGamma,contBinom,contExpon,contWide,contNarrow,contOutlier,contcor1,contcor2,facGender,facExperim,facFive,facFifty,facOutlier,debString,debMiss1,debMiss30,debMiss80,debMiss99,debBinMiss20,debNaN,debNaN10,debInf,debCollin1,debCollin2,debCollin3,debEqual1,debEqual2,debSame,unicode
```

Comment:
you don't need to add this file, using "debug.csv" or "test.csv" as data set argument for jaspTools should suffice.